atlas-chart/src/main/scala/com/netflix/atlas/chart/util/PngImage.scala

```diff
@@ -87,12 +89,22 @@ object PngImage {
 
   def userError(imgText: String, width: Int, height: Int): PngImage = {
     val userErrorYellow = new Color(0xFF, 0xCF, 0x00)
-    error(imgText, width, height, "USER ERROR:", Color.BLACK, userErrorYellow)
+    val clampedWidth = math.min(width, GraphConstants.MaxWidth)
+    val clampedHeight = math.min(height, GraphConstants.MaxHeight)
+    
+    val dimensionWarning = if (width > GraphConstants.MaxWidth || height > GraphConstants.MaxHeight) {
+      s" Image dimensions clamped to ${clampedWidth}x${clampedHeight} (max: ${GraphConstants.MaxWidth}x${GraphConstants.MaxHeight})."
+    } else ""
```

Comment:
Not sure we need to append this detail to the error images. They will see it as a warning on the regular graph once they fix whatever error is being shown. Adding details about restricting the image size to the error could just cause confusion and distract from the issue that led to the error.

Reply:
Yah I can take it out if you like. My thoughts were about surfacing what value was expected. No problem, will remove.